### PR TITLE
'create_role' method for BioBlend

### PR DESCRIPTION
**Pull request by ashvark**
This PR will enable the admin users to create a new role using Bioblend.

### CODE REVIEW

**Comment by nsoranzo** [comment]
Thanks a lot, looks good to me. It would be nice to add a test to `bioblend/_tests/TestGalaxyRoles.py`, if you have time.

**Comment by ashvark** [comment]
@nsoranzo Thanks for your comments. I will try to add a test for this method.

**Comment by ashvark** [comment]
@nsoranzo . I made an attempt to create a test for create_role method. But I did not manage to test it. Can you tell me how to check whether this test_create_role method is working fine ?

File: bioblend/_tests/TestGalaxyRoles.py [hunk 1]
```diff
@@ -4,13 +4,30 @@
...
 from . import GalaxyTestBase
 
 
 class TestGalaxyRoles(GalaxyTestBase.GalaxyTestBase):
 
+    def setUp(self):
+        super(TestGalaxyRoles, self).setUp()
+        self.name = 'test_%s' % uuid.uuid4().hex
+        self.description = 'automated test role'
+        self.role = self.gi.roles.create_role(self.name)[0]
```
Comment (mvdbeek) [inline bioblend/_tests/TestGalaxyRoles.py]: You need to provide name and description here.

Comment (ashvark) [inline bioblend/_tests/TestGalaxyRoles.py]: Yes you are right. Thank you.

**Comment by mvdbeek** [comment]
You can run the tests using nosetest, `nosetests -v bioblend/_tests/TestGalaxyRoles.py` should work. Before running the test you also need to provide the environment variables `BIOBLEND_GALAXY_URL` and `BIOBLEND_GALAXY_API_KEY`. Alternatively that would be taken care of using `run_bioblend_tests.sh -t bioblend/_tests/TestGalaxyRoles.py`

**Comment by ashvark** [comment]
Thanks @mvdbeek  I managed to test my code now and it works fine.

**Comment by nsoranzo** [comment]
Thanks a lot @ashvark!

**Pull request merged**